src/content/contribute/writing-a-loader.md

```diff
@@ -361,13 +365,16 @@ And now, finally, we can write our test and add an npm script to run it:
 __test/loader.test.js__
 
 ```js
+/**
+ * @jest-environment node
```

Comment:
`jest-environment` defaults to `jsdom` https://jestjs.io/docs/en/configuration#testenvironment-string, which is not we want here.